### PR TITLE
Update versions.

### PR DESCRIPTION
**Pull request by paulwang007**
Signed-off-by: Wang, Paul <paul.wang@rakuten.com>

### CODE REVIEW

File: versions/versions.gradle [hunk 1]
```diff
@@ -7,27 +7,27 @@
...
     java      : '1.7',
-    kotlin    : '1.2.0',
+    kotlin    : '1.3.21',
 
     // Android plugins, tools, SDK and libraries
     android   : [
         libraries: [
-            playServices: '11.0.4',
-            support     : '26.0.0',
+            playServices: '16.0.0',
```
Comment (corycaywood) [inline versions/versions.gradle]: A couple of things regarding play services:
1. Play services should no longer have the same version for all of it's libraries ([see here](https://developers.google.com/android/guides/versioning)). And actually, some of the libraries only have a `v16.0.1` and no longer have `v16.0.0` available. 
2. Some SDKs that are using this value will break if updated past `11.0.4` for their play-services libraries.

So I think we should either leave it at `11.0.4` and then stop using it (just leave it up to each SDK to implement the version) or completely remove `playServices` from this repo and then possibly replace it with versions for the commonly used individual play-services libraries. What do you think?

Comment (paulwang007) [reply versions/versions.gradle]: Nice catch, I will update it.

Comment (paulwang007) [reply versions/versions.gradle]: @corycaywood Reverted back to 11.0.4

**Comment by corycaywood** [comment]
Looks like there's a conflict now, so you will need to rebase on master and resolve it first.

**Comment by paulwang007** [comment]
@corycaywood for some reason after I rebased, PR can't be opened anymore.

**Comment by paulwang007** [comment]
@corycaywood conflict fixed.

**Comment by paulwang007** [comment]
@corycaywood For some reason this commit fell thru the crack.

File: versions/versions.gradle (hunk 1 above)
Comment (NemoOudeis) [reply versions/versions.gradle]: it's still `16.0.0`

File: versions/versions.gradle [hunk 2]
```diff
@@ -7,27 +7,27 @@
...
+            support     : '28.0.0',
```
Comment (NemoOudeis) [inline versions/versions.gradle]: Similar to play services updating the support library to 28.0.0 creates incompatibilities. For the SDKs we will probably stay on 26 until we switch to androidx - or what do you think @corycaywood ?

File: versions/versions.gradle [hunk 3]
```diff
@@ -7,27 +7,27 @@
...
     ],
 
     // Third party dependencies of our SDK
     thirdParty: [
-        autoParcelGson: '0.2',
-        gson          : '2.8.1',
-        threetenbp    : '1.3.6',
+        autoParcelGson: '1.0.3',
+        gson          : '2.8.5',
+        threetenbp    : '1.3.8',
```
Comment (NemoOudeis) [inline versions/versions.gradle]: @corycaywood should we remove these? i don't think we need a central registry of these dependencies that are used sometimes and adhere to semantic versioning.

Comment (corycaywood) [reply versions/versions.gradle]: Yeah, I think we can remove them. We also don't use them in many SDKs anymore.

File: versions/versions.gradle (hunk 2 above)
Comment (corycaywood) [reply versions/versions.gradle]: Yeah, good point. We should probably leave it at 26 and start switching to androidx libraries.

**Pull request closed without merging**